ant-networking/Cargo.toml

```diff
@@ -20,6 +20,7 @@ upnp = ["libp2p/upnp"]
 
 [dependencies]
 aes-gcm-siv = "0.11.1"
+alloy = { version = "0.5.3", default-features = false, features = ["std", "reqwest-rustls-tls", "provider-anvil-node", "sol-types", "json", "signers", "contract", "signer-local", "network"] }
```

Comment:
Was this only added for the `U256` type? Cause the type is in `ant_evm::common::U256` already 🤔

Reply:
^ I'll do that in my merge commit for PR # 2519